js/state/syncUtil.js

```diff
@@ -43,7 +43,7 @@ module.exports.siteSettingDefaults = {
   shieldsUp: true,
   adControl: 1,
   cookieControl: 0,
-  safeBrowsing: true,
+  safeBrowsingControl: 0,
```

Comment:
this will require changes to the brave/sync library and cross-platform sync implementations. i think we should avoid changes to sync for now, since other platforms don't support Google safebrowsing yet.

Reply:
FYI the sync changes would be needed in https://github.com/brave/sync/blob/staging/lib/api.proto#L68

Reply:
https://github.com/brave/sync/pull/198

Reply:
per comment on the sync PR, i think we should just stub `safeBrowsing` to be `undefined` for sync.